applications/StructuralMechanicsApplication/CMakeLists.txt

```diff
@@ -196,4 +196,6 @@ install(TARGETS KratosStructuralMechanicsCore DESTINATION libs )
 install(TARGETS KratosStructuralMechanicsApplication DESTINATION libs )
 
 # Add to the KratosMultiphisics Python module
-install(FILES "${CMAKE_CURRENT_SOURCE_DIR}/StructuralMechanicsApplication.py" DESTINATION KratosMultiphysics )
+install(DIRECTORY DESTINATION "KratosMultiphysics/StructuralMechanicsApplication")
```

Comment:
I am not sure if this works in Win (using `/` for separating the path)
=> if not I think it can be easily fixed

Reply:
Usually you are safe with "/". It is also a valid separator for windows.

Reply:
so this is not any longer in 

KratosMultiphysics/applications/StructuralMechanicsApplication? 

this means that you can only make installation outside of where you compile?

Reply:
I don’t understand, can you please explain in more detail what you mean?

Reply:
right now on my computer i normally keep the python files in the same directory as they normally are. This is useful since when i change a file i do not need to reinstall.

I am asking if this will change after this PR or if the python files will be left where they are

Reply:
clarifying on my point, it may even be good if this is changed, since it may be the first step towards propoer python packaging of Kratos...however the need of installing every time a python file is modified is a little annoying, which is the reason for which i ask

Reply:
Ah now I understand
Answer: the current behavior is NOT changed, i.e. you DON'T have to reinstall after changing python files
Only the folder-structure changes a bit in the `KratosMultiphysics` folder
from:
~~~
KratosMultiphysics
    - __init__.py
    - StructuralMechanicsApplication.py
    - FluidDynamicsApplication.py
~~~

to:
~~~
KratosMultiphysics
    - __init__.py
    - StructuralMechanicsApplication
        __init__.py # this is the "StructuralMechanics.py"-file
    - FluidDynamicsApplication.py
~~~

Reply:
oh, i see.

thx!